server/api/utils/clients/async_nuclio.py

```diff
@@ -102,3 +178,18 @@ def _handle_error_response(
         self._logger.warning("Request to nuclio failed. Reason:", **log_kwargs)
 
         mlrun.errors.raise_for_status(response, error_message)
+
+    def _resolve_nuclio_api_gateway(
```

Comment:
Not really resolving anything. perhaps Enriching
```suggestion
    def _enrich_nuclio_api_gateway(
```